src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/LambdaCompiler.Statements.cs

```diff
@@ -152,6 +152,28 @@ private void EmitLoopExpression(Expression expr)
         private void EmitSwitchExpression(Expression expr, CompilationFlags flags)
         {
             SwitchExpression node = (SwitchExpression)expr;
+            
+            if (node.Cases.Count == 0)
+            {
```

Comment:
I would have thought on first blush that this was a matter of changing LightCompiler.cs at line 2937 and StackSpiller.Generated.cs (generated by what?) at line 189 to branch on `.CanReduce`. And that seems to be the case with StackSpiller anyway, but I'm not a all clear on how to test LightCompiler.cs, which makes me obviously much less confident with this than once I was.